lib/hyperion/superion.rb

```diff
@@ -0,0 +1,102 @@
+require 'hyperion'
+
+module Superion
+
+  # @param [RestRoute] route The route to request
+  # @option opts [Object] :body The payload to POST/PUT. Usually a Hash or Array.
+  # @option opts [Hash<predicate, transformer>] :also_handle Additional handlers to
+  #   use besides the default handlers. A predicate is an integer HTTP code, an
+  #   integer Range of HTTP codes, a HyperionResult::Status enumeration value,
+  #   or a predicate proc. A transformer is a procedure which accepts a
+  #   HyperionResult and returns the final value to return from `request`
+  # @option opts [Proc] :render A transformer, usually a proc returned by
+  #   `as` or `as_many`. Only called on HTTP 200.
+  # @yield [rendered] Yields to allow an additional transformation.
+  #   Only called on HTTP 200.
+  def request(route, opts={}, &project)
+    guard_param(route, RestRoute, 'a RestRoute')
+    guard_param(opts, Hash, 'an options hash')
+
+    body = opts[:body]
+    additional_handler_hash = opts[:also_handle] || {}
+    render = opts[:render] || Proc.identity
+    project = project || Proc.identity
+
+    Hyperion.request(route, body) do |result|
+      all_handlers = [hash_handler(additional_handler_hash),
+                      handler_from_including_class,
+                      built_in_handler(project, render)]
+
+      all_handlers.each { |handlers| handlers.call(result) }
+      fallthrough(result)
+    end
+  end
+
+  # PW: deprecate
+  def missing
+    proc do |result|
+      result.body.errors.detect(:code, ErrorInfo::Code::MISSING)
+    end
+  end
```

Comment:
It is no longer necessary, now that you can use `ErrorInfo::Code::MISSING` directly as a condition to handle, just like you can with `HyperionResult::Status` values.

https://github.com/indigo-biosystems/hyperion/blob/master/spec/lib/hyperion_spec.rb#L125
(works for the hash-style "request" handlers too)